README.md

```diff
@@ -1,101 +1,91 @@
 
- ` error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.`
- 
-Change your current working directory to  `amazon-kinesis-video-streams-webrtc-sdk-ios/Swift` and run the following in the command line:
-```
-$pod cache clean --all
-$pod install
-```
+#### Run the iOS Sample Application
+Building the iOS sample application installs the AWSKinesisVideoWebRTCDemoApp on your iOS device. Using this app, you can verify live audio/video streaming between mobile, web and IoT device clients (camera). The procedure below describes some of these scenarios. 
 
+Complete the following steps:
 
-Change your current working directory to  `amazon-kinesis-video-streams-webrtc-sdk-ios`
-```
-$git checkout Swift/Pods/AWSCore/AWSCore/Service/AWSService.m
-```
+1.    On your iOS device, open AWSKinesisVideoWebRTCDemoApp and login using the AWS user credentials from Set Up an AWS Account and Create an Administrator. (Note: Cognito settings can be tuned through your Cognito User Pool in the AWS management Console)
+2.    On successful sign-in, the channel configuration view is displayed where the **channel-name, client-id (optional) and region-name** have to be configured. 
 
-Run the `Build` again.
- 
+##### Note
+*    Ensure that in all the cases described below, both the client applications use the same signaling channel name, region, viewer-id/client-id and the AWS account id.
+*    Please note that a master should be started first before the viewer connects to it.
 
+#####    Peer to Peer Streaming between two iOS devices: master and viewer:
+*    Start one iOS device in master mode for starting a new session using a channel name (e.g. demo). Remote peer will be joining as viewer to this master.
+*    Currently, there can be only one master for a channel at any given time.
+*    Use another iOS device to connect to the same channel name (started up in the above step set up as a master) in viewer mode. This will connect to an existing session (channel) where a master was connected previously.
 
-#### Run the iOS Sample Application
-Building the iOS sample application installs the AWSKinesisVideoWebRTCDemoApp on your iOS device. Using this app, you can verify live audio/video streaming between mobile, web and IoT device clients (camera). The procedure below describes some of these scenarios. Complete the following steps:
-1.	On your iOS device, open AWSKinesisVideoWebRTCDemoApp and login using the AWS user credentials from Set Up an AWS Account and Create an Administrator. (Note: Cognito settings can be tuned through your Cognito User Pool in the AWS management Console)
-2.	On successful sign-in, the channel configuration view is displayed where the **channel-name, client-id (optional) and region-name* have to be configured. 
+#####    Peer to Peer Streaming between Embedded SDK master and iOS device:
+  *    Run KVS WebRTC embedded SDK (in C) in master mode on a camera device.
```

Comment:
Put a link https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c on KVS WebRTC embedded SDK would be good.